src/components/SpaceProposalBoostWinnersModal.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+import { BoostSubgraph } from '@/helpers/boost/types';
+import { formatUnits } from '@ethersproject/units';
+
+const props = defineProps<{
+  open: boolean;
+  boost: BoostSubgraph;
+  winners: string[];
+  prize: string;
+}>();
+
+defineEmits(['close']);
+
+const { web3Account } = useWeb3();
+const { formatNumber, getNumberFormatter } = useIntl();
+const { loadProfiles, profiles } = useProfiles();
+
+const searchInput = ref('');
+const loading = ref(false);
+
+const formattedPrize = computed(() => {
+  if (!props.prize) return '0';
+  const formattedPrize = formatUnits(props.prize, props.boost.token.decimals);
+  return formatNumber(
+    +formattedPrize,
+    getNumberFormatter({ maximumFractionDigits: 8 }).value
+  );
+});
+
+const sortedWinners = computed(() => {
+  if (!props.winners) return [];
+  return props.winners
+    .filter(winner =>
+      winner.toLowerCase().includes(searchInput.value.toLowerCase())
+    )
+    .sort((a, b) => {
+      if (a.toLowerCase() === web3Account.value.toLowerCase()) return -1;
+      if (b.toLowerCase() === web3Account.value.toLowerCase()) return 1;
+      return 0;
+    });
+});
+
+watch(
+  () => props.winners,
+  () => {
+    if (!props.winners) return;
+    loadProfiles(props.winners);
+  },
+  { immediate: true }
+);
+</script>
+
+<template>
+  <!-- TODO: Show link to randomness docs  -->
+  <TuneModal :open="open" @close="$emit('close')">
+    <TuneModalTitle as="h4" class="flex items-center gap-1 m-3 mb-2">
+      Winners
+      <BaseCounter v-if="winners" :counter="winners.length" />
+    </TuneModalTitle>
+    <BaseSearch
+      v-model="searchInput"
+      placeholder="Search by address"
+      modal
+      class="!pl-3"
+    />
```

Comment:
The `x` button is not aligned with the rest

![Screenshot 2024-03-06 at 23 49 34](https://github.com/snapshot-labs/snapshot/assets/495709/ea5fe664-820a-4740-9492-6db8b8da23bf)